apps/protocol-frontend/src/components/ContributionsTable.tsx

```diff
@@ -158,16 +161,13 @@ const ContributionsTable = ({
                     handleEditContributionFormModal(row.original.id)
                   }
                 />
-                <IconButton
-                  icon={<FiTrash2 fontSize="1rem" />}
-                  variant="ghost"
-                  color="gray.800"
-                  disabled={
-                    row.original.user.id !== userData?.id ||
-                    row.original.status === 'minted'
-                  }
-                  aria-label="Delete Contribution"
-                />
+                <DeleteContributionDialog 
```

Comment:
Hmm, not sure why we'd want to split this off from the Table itself
The delete icon button should be able to call the Dialog

Reply:
Thanks for the feedback. I made some changes based on your comments above (letting the icon button call the Dialog)